docs/release-notes/NuGet-6.1.md

```diff
@@ -101,6 +101,16 @@ NuGet distribution vehicles:
 
 **[List of all issues fixed in this release - 6.1](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvNjY5ODY)**
 
+## Known issues
+
+### dotnet nuget push raises `error: File does not exist ...` exception. - [#10601](https://github.com/NuGet/Home/issues/11601)
+
+#### Issue
+Previously in order to use `-n|--no-symbols` and `-d|--disable-buffering` options with `dotnet nuget push`  command requires passing unnecessary passing of random value after it, even though it's not mentioned in our doc. Removal of this unnecessary value can break your script by throwing exception with `error: File does not exist ...` even though actual push operation was successful.
```

Comment:
Remove the space after `dotnet nuget push`. Get rid of the "it's not mentioned in our doc".

Try to be directive of the issue.